docs/fragments/lib/android-sdk.md

```diff
@@ -1,4 +1,13 @@
-## AWS Mobile SDK for Android
-
+**AWS Mobile SDK for Android**  
 This guide shows how to build an app using AWS Mobile SDK for Android and the Amplify CLI toolchain.
-To use our new, preview developer experience with new Amplify Libraries for iOS, [click here](/lib).
+To use our new, preview developer experience with new Amplify Libraries for Android, [click here](~/lib/lib.md).
```

Comment:
Change the language to: "Click here to use our new preview developer experience..."